include/Dialect/Secret/Transforms/Passes.td

```diff
@@ -0,0 +1,15 @@
+#ifndef INCLUDE_DIALECT_SECRET_TRANSFORMS_PASSES_TD_
+#define INCLUDE_DIALECT_SECRET_TRANSFORMS_PASSES_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def SecretForgetSecrets : Pass<"secret-forget-secrets"> {
```

Comment:
"Forgetting" secrets to me sounds like something that removes secret values, rather than the secret annotation. How about naming this "drop-secret-status" or something along those lines?

Reply:
I was thinking of the ["forgetful functor"](https://en.wikipedia.org/wiki/Forgetful_functor) but maybe that is too big of a stretch.

I like the slightly more specific "drop-secret-to-cleartext" or "interpret-secret-as-cleartext", will update it on Monday.